### PR TITLE
Changes the peer review requested

### PR DESCRIPTION
**Pull request by steget**

### CODE REVIEW

File: src/main/java/storagesystem/model/StoreIT.java [hunk 1]
```diff
@@ -64,16 +67,18 @@ private static void mockData() {
...
-        organisations.add(data);
-        setCurrentOrganisation(informationsteknik);
-
         tempTeam.addItemToInventory(mockItem);
         tempTeam.addItemToInventory(mockItem2);
     }
 
-    public static void setCurrentUser(User currentUser) {
-        StoreIT.currentUser = currentUser;
+    public static void setCurrentUser(String username) {
```
Comment (Sponken) [inline src/main/java/storagesystem/model/StoreIT.java]: Javadoc? Doesn't really seam necessary, but at the same time we don't have an explanation of currentUser

File: src/main/java/storagesystem/model/StoreIT.java [hunk 2]
```diff
@@ -64,16 +67,18 @@ private static void mockData() {
...
+        for (User u :
+                currentOrganisation.getUsers()) {
+            if (u.getName().equals(username)) {
+                currentUser = u;
+                break;
+            }
+        }
     }
 
     public static void setCurrentOrganisation(Organisation currentOrganisation) {
```
Comment (Sponken) [inline src/main/java/storagesystem/model/StoreIT.java]: Javadoc? Doesn't really seam necessary, but at the same time we don't have an explanation of currentUser

File: src/main/java/storagesystem/model/StoreIT.java [hunk 3]
```diff
@@ -106,4 +112,49 @@ public static Organisation findOrganisation(String organisationName) throws NoSu
...
+     * @param password           Password for login
+     * @param description        Some information the user provides about themself
+     * @param contactInformation Some sort of way to contact the User, preferably phone/mail
+     */
+    public static void createUser(String name, String password, String description, String contactInformation) {
+        currentOrganisation.addUser(new User(name, password, description, contactInformation));
+    }
+
+    /**
+     * Compares with params if there is such a user
```
Comment (Sponken) [inline src/main/java/storagesystem/model/StoreIT.java]: Unclear. Compares what with params?

File: src/main/java/storagesystem/model/StoreIT.java [hunk 4]
```diff
@@ -106,4 +112,49 @@ public static Organisation findOrganisation(String organisationName) throws NoSu
...
+     *
+     * @param username Name to find in the database
```
Comment (Sponken) [inline src/main/java/storagesystem/model/StoreIT.java]: Not in database, in organisation

File: src/main/java/storagesystem/model/User.java [hunk 5]
```diff
@@ -27,24 +25,11 @@ public User(String name, String password, String description, String contactInfo
...
     public User(String name) {
-        this(name, "", "");
-    }
-
-    private User(User userToCopy) {
-        this.name = userToCopy.name;
-        this.description = userToCopy.description;
-        this.contactInformation = userToCopy.contactInformation;
-        this.ID = userToCopy.ID;
+        this(name, "", "", "");
```
Comment (Sponken) [inline src/main/java/storagesystem/model/User.java]: Should you be able to create a user like this?

File: src/main/java/storagesystem/model/StoreIT.java [hunk 6]
```diff
@@ -106,4 +112,49 @@ public static Organisation findOrganisation(String organisationName) throws NoSu
...
+     * @param password Password supposed to match with the username
+     * @return {@code True} if the username and password matches, {@code False} if there is no match
+     */
+    public static boolean doesLoginMatch(String username, String password) {
+        for (User user :
+                currentOrganisation.getUsers()) {
+            if (user.getName().equals(username) && user.getPassword().equals(password)) {
```
Comment (Kviick) [inline src/main/java/storagesystem/model/StoreIT.java]: Username to lowercase

File: src/main/java/storagesystem/model/StoreIT.java [hunk 7]
```diff
@@ -106,4 +112,49 @@ public static Organisation findOrganisation(String organisationName) throws NoSu
...
+
+    /**
+     * Checks in the selected organisation if there is an user with the name currently written in the Username textfield
+     *
+     * @return If the written username exists within the selected Organisation
+     */
+    public static boolean doesUserExist(Organisation organisationToSearch, String name) {
+        for (User user :
+                organisationToSearch.getUsers()) {
+            if (user.getName().equals(name)) {
```
Comment (Kviick) [inline src/main/java/storagesystem/model/StoreIT.java]: check lowercase

File: src/main/java/storagesystem/model/User.java (hunk 5 above)
Comment (steget) [reply src/main/java/storagesystem/model/User.java]: Mostly just for testing purposes, could be deleted

File: src/main/java/storagesystem/model/StoreIT.java (hunk 1 above)
Comment (steget) [reply src/main/java/storagesystem/model/StoreIT.java]: Self-explanatory imo

**Pull request merged**